examples/external-dns/terraform.tfvars.example

```diff
@@ -0,0 +1,9 @@
+dns_create_record    = "true"
+node_count           = "value must be either 1 or 2"
+network              = "projects/xxxxxxx/global/networks/xxxxxxx"
+subnetwork           = "xxxxxxxxx"
```

Comment:
Instead of `xxxxxxx`, can you put what a "Pre-existing subnetwork self link" looks like? A dummy value?